cs.json

```
@@ -0,0 +1,347 @@
+{
+    "name" : "Čeština",
+    "live" : true,
+    "translations" : {
+        "meta" : {
+            "thForDate" : true,
```

```suggestion
            "thForDate" : false,
```
This option applies only to English or English-like languages. So we don't need to enable it, otherwise `th` would be appended to the day of a date.